diagnostic.md

```diff
@@ -150,7 +177,9 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+the first div will be on the top left.
+the second will be on the to right.
+the third will be under the first (and left too)
```

Comment:
```.divOne``` will be in the top-right corner;
```.divTwo``` will be in the top-left corner, with .divThree immediately to its right;
```.divFour``` will be underneath ```.divTwo```, and ```.divFive``` will sit immediately to the right of ```.divFour``` (under ```.divThree```).